Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -62,27 +63,6 @@ extension Issue {
   /// Use this function if, while running a test, an issue occurs that cannot be
   /// represented as an expectation (using the ``expect(_:_:sourceLocation:)``
   /// or ``require(_:_:sourceLocation:)-5l63q`` macros.)
-  @discardableResult public static func record(
-    _ comment: Comment? = nil,
-    sourceLocation: SourceLocation = #_sourceLocation
-  ) -> Self {
-    record(comment, severity: .error, sourceLocation: sourceLocation)
-  }
-
-  /// Record an issue when a running test fails unexpectedly.
-  ///
-  /// - Parameters:
-  ///   - comment: A comment describing the expectation.
-  ///   - severity: The severity of the issue.
-  ///   - sourceLocation: The source location to which the issue should be
-  ///     attributed.
-  ///
-  /// - Returns: The issue that was recorded.
-  ///
-  /// Use this function if, while running a test, an issue occurs that cannot be
-  /// represented as an expectation (using the ``expect(_:_:sourceLocation:)``
-  /// or ``require(_:_:sourceLocation:)-5l63q`` macros.)
-  @_spi(Experimental)
   @discardableResult public static func record(
     _ comment: Comment? = nil,
     severity: Severity,
```

Comment:
Now, this should gain a default value of `.error`. That couldn't be added until now, when the API is promoted, because it would've been ambiguous with the prior overload
```suggestion
    severity: Severity = .error,
```